charts/ps-db/templates/cluster.yaml

```diff
@@ -416,6 +435,26 @@ spec:
       {{- end }}
       {{- if $router.gracePeriod }}
       gracePeriod: {{ $router.gracePeriod }}
+      {{- end }}
+      {{- if $router.env }}
+      env:
+{{ $router.env | toYaml | indent 8 }}
```

Comment:
$router.env all other items are'nt working for router. Please check the template.

Reply:
https://github.com/percona/percona-helm-charts/pull/669/commits/17dd0a46c8916a450a8bbaeffdf76d86eb0be579